index.html

```diff
@@ -181,6 +181,7 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script src="js/my.js"></script>
```

Comment:
перенеси прям весь файл в index.html, чтобы линтер не подключался, в этом задании, похоже он не нужен

```suggestion
<script>
  var fireballSize = 22;
  var wizardSpeed = 3;
  var wizardWidth = 70;

  var getFireballSpeed = function (left) {
    if (left) {
      return 5;
    }

    return 2;
  };

  // Вычесляем высоту мага
  var getWizardHeight = function () {
    var proportionalHeight = 1.337 * wizardWidth;
    return proportionalHeight;
  };

  // Положение мага по горизонтали
  var getWizardX = function (width) {
    var wizardX = width / 3 + wizardWidth / 2;
    return wizardX;
  };

  // Положение мага по вертикали
  var getWizardY = function (height) {
    var wizardY = height * 2 / 60;
    return wizardY;
  };
</script>
```